src/transaction/log_page_buffer.c

```diff
@@ -6092,6 +6096,36 @@ logpb_remove_archive_logs_exceed_limit (THREAD_ENTRY * thread_p, int max_count)
 		  assert (false);
 		}
 	    }
+
+	  /* check flashback */
+	  if (flashback_check_time_to_remove_archive ())
```

Comment:
This mechanism makes a timing problem. I mean that every request should be completed within the threshold. But, the archive volumes should never be deleted during the request is processed. So, you just consider the time between a request and the next request and abnormal termination of connection.

Reply:
Yes, I will consider the intervals between requests and flashback state.